fadecandy_driver/src/fadecandy_driver.cpp

```diff
@@ -44,21 +44,29 @@ constexpr int USB_VENDOR_ID = 0x1d50;
 constexpr int USB_ENDPOINT = 1;
 constexpr int INTERFACE_NO = 0x01;
 
-std::string FadecandyDriver::connect()
+FadecandyDriver::FadecandyDriver()
 {
-  // Find usb device.
-  libusb_device_descriptor fadecandy_device_descriptor = findUsbDevice();
-
   uint r = libusb_init(&context_);
```

Comment:
uint is always larger than `0`, this does not make sense. `libusb_init` returns an `int` @jadhm